code/modules/shuttle/mini_dropship.dm

```diff
@@ -169,6 +169,14 @@
 	s2.start()
 	damaged = TRUE
 	remove_eye_control(ui_user)
+	if(fly_state == SHUTTLE_IN_ATMOSPHERE && last_valid_ground_port)
```

Comment:
check for destination as well (should be fly_state), if you are already going up on ship you don't want to go on ground

Reply:
I didn't really think about it, but after testing it is intended.